octobot_trading/exchanges/implementations/ccxt_exchange_commons.py

```diff
@@ -0,0 +1,285 @@
+                                                   check_completeness=check_completeness, **kwargs)
+
+    async def get_open_orders(self, symbol: str = None, since: int = None, limit: int = None,
+                              check_completeness: bool = None, **kwargs: dict) -> list:
+        return await self.connector.get_open_orders(symbol=symbol, since=since, limit=limit,
+                                                    check_completeness=check_completeness, **kwargs)
+
+    async def get_closed_orders(self, symbol: str = None, since: int = None, limit: int = None,
+                                check_completeness: bool = None, **kwargs: dict) -> list:
+        return await self.connector.get_closed_orders(symbol=symbol, since=since, limit=limit,
+                                                      check_completeness=check_completeness, **kwargs)
+
+    async def get_my_recent_trades(self, symbol: str = None, since: int = None, limit: int = None,
+                                   check_completeness: bool = None, **kwargs: dict) -> list:
+        return await self.connector.get_my_recent_trades(symbol=symbol, since=since, limit=limit,
+                                                         check_completeness=check_completeness, **kwargs)
+
+    async def get_recent_trades(self, symbol: str, limit: int = 50,
+                                check_completeness: bool = None, **kwargs: dict) -> typing.Optional[list]:
+        return await self.connector.get_recent_trades(symbol=symbol, limit=limit, check_completeness=check_completeness,
+                                                      **kwargs)
+
+    async def cancel_order(self, order_id: str, symbol: str = None, **kwargs: dict) -> bool:
+        return await self.connector.cancel_order(symbol=symbol, order_id=order_id, **kwargs)
+
+    def get_trade_fee(self, symbol, order_type, quantity, price, taker_or_maker):
+        return self.connector.get_trade_fee(symbol, order_type, quantity, price, taker_or_maker)
+
+    def get_fees(self, symbol):
+        return self.connector.get_fees(symbol)
+
+    def get_pair_from_exchange(self, pair) -> str:
+        return self.connector.get_pair_from_exchange(pair)
+
+    def get_split_pair_from_exchange(self, pair) -> typing.Tuple[str, str]:
+        return self.connector.get_split_pair_from_exchange(pair)
+
+    def get_exchange_pair(self, pair) -> str:
+        return self.connector.get_exchange_pair(pair)
+
+    def get_pair_cryptocurrency(self, pair) -> str:
+        return self.connector.get_pair_cryptocurrency(pair)
+
+    def get_default_balance(self):
+        return self.connector.get_default_balance()
+
+    def get_rate_limit(self):
+        return self.connector.get_rate_limit()
+
+    async def switch_to_account(self, account_type: enums.AccountTypes):
+        return await self.connector.switch_to_account(account_type=account_type)
+
+    def parse_balance(self, balance):
```

Comment:
I don't understand why we still need parse methods here

Reply:
parse balance does not have its own parser yet

Reply:
Oh yes that's right. Should we create it?

Reply:
I'll do that later once this version is finished